src/prometheus_alert_rules/cpu.rules

```diff
@@ -0,0 +1,27 @@
+groups:
+- name: HostCPU
+  rules:
+
+  # Compute number of cores
+  - record: node_cpu_cores_total
+    expr: count(node_cpu_seconds_total{mode="idle"}) without (cpu,mode)
```

Comment:
This is probably the first time a PR introduces recording rules to our set of built-in rules.
This is a good opportunity to discuss best practices.
1. [Official docs](https://prometheus.io/docs/practices/rules/#recording-rules) recommend naming in the general form `level:metric:operations`. I think in this case it would be `instance:node_cpu_seconds:count`.
2. Recording rules are meant for massively recurring expensive operations. Seems like here it is used as an alias to simplify other _alert_ expressions. Not sure where the best tradeoff lies.

@rbarry82 @simskij

Reply:
Calculating it through a recording rule also means another time series will be created. In this particular case, that time series will be of negligible size, so it does not really matter much. 

Will you want to use this recorded metric in your dashboards? If yes, it might make sense. If not, might be just as good to inline it.

Reply:
Since all of these expressions have been deleted, I'll go ahead and resolve the conversations.